tests/Sdk/Tests/Core/Application/Service/CommandExecutorTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Tests\Core\Application\Service;
+
+use Codeception\Test\Unit;
+use PHPUnit\Framework\MockObject\Rule\InvocationOrder;
+use SprykerSdk\Sdk\Core\Appplication\Dependency\AfterCommandExecutedAction\AfterCommandExecutedActionInterface;
+use SprykerSdk\Sdk\Core\Appplication\Service\CommandExecutor;
+use SprykerSdk\SdkContracts\CommandRunner\CommandRunnerInterface;
+use SprykerSdk\SdkContracts\Entity\CommandInterface;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+
+/**
+ * @group Sdk
+ * @group Core
+ * @group Application
+ * @group Service
+ * @group CommandExecutorTest
+ */
+class CommandExecutorTest extends Unit
+{
+    /**
+     * @return void
+     */
+    public function testExecute(): void
+    {
+        $context = $this->createContextMock();
```

Comment:
The same as here https://github.com/spryker-sdk/sdk/pull/61/files#r786651379
Please look through your tests and add comments